website/server/controllers/api-v3/groups.js

```diff
@@ -1068,6 +1069,36 @@ api.inviteToGroup = {
 
     Group.validateInvitations(uuids, emails, res);
 
+    // If party, check the limit of members
+    if (group.type === 'party') {
+      let memberCount = 0;
+
+      // Counting the members that already joined the party
+      memberCount += group.memberCount;
+
+      // Count how many invitations currently exist in the party
+      let query = {};
+      query['invitations.party.id'] = group._id;
+      let groupInvites = await User
+        .find(query)
+        .select('_id')
+        .exec();
+      memberCount += groupInvites.length;
+      
+      // Counting the members that are going to be invited by email and uuids
+      if (uuids) {
```

Comment:
Do you mean that I could put all my code that validates the size of party inside ``Group.validateInvitations``?